alephmarcreader/abstractalephmarcreader.py

```diff
@@ -7,39 +7,67 @@
 
 class AbstractAlephMarcReader(ABC):
 
-    class Person:
+    class Correspondent(ABC):
+        """
+        Represents an abstract correspondent.
+        Both Persons and Organisations can be Correspondents.
+        :param str|False name: the name of the correspondent, if any.
+        :param str|False gnd: the GND of the correspondent, otherwise 'no_GND', if any.
+        :param str|False roles: the role of the correspondent (author etc.), if any.
+
+        """
+
+        CORRESPONDENT_TYPE_PERSON = 'Person'
+        CORRESPONDENT_TYPE_ORGANISATION = 'Organisation'
+
+        def __init__(self, name, gnd, roles):
+            self.name = name
+            self.gnd = gnd
+            self.roles = roles
+
+        @abc.abstractmethod
+        def get_type(self):
+            """
+            Gives the type of Correspondent.
+            :return str: Type of correspondent, either 'person' or 'organisation'.
+            """
+            pass
```

Comment:
Is the `pass` necessary? The method is abstract and does not need an implementation.

Reply:
I think it needs that... I may have googled "python abstract class" and have found something like [this](https://www.python-course.eu/python3_abstract_classes.php), where they use `pass`.

But I can't say for sure, that it's necessary. What I can say for sure, though, is that it works as it is now.

Reply:
I think you are right, we used it for other abstract methods too. I think I simply coded too much TypeScript ...